extensions/amp-bento/0.1/amp-bento.css

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Special elements that require FOUC. */
+
+@import 'amp-accordion.css';
+@import 'amp-social-share.css';
+
+
+/* Common boilerplate */
```

Comment:
Should this include the
```
:not(:defined) > * {
  visibility: hidden;
}
```
?

Reply:
With enumerated or per-element approach, I'd say doing a blanket `:not(:defined) {}` would be harmful, because of three reasons:

1. This would collide with new HTML#X elements that are not yet supported on some platform. E.g. a hypothetical new `<popup>` element.
2. Sometimes devs do use special-need undefined tags. E.g. "my-section".
3. There might be another web components library we'd collide with. Not super likely, but still.